Changelog: Add DM 1.0 GA (#271)
pingcap/dm: CHANGELOG.md

All notable changes to this project will be documented in this file.

## [1.0.0] 2019-09-06

### v1.0.0 What's New

#### Improvements
- Add auto recovery framework
- Make task retryable when encounters a database driver error
- Make task retryable when encounters network issue
- Improve the DDL compatibility in DM

#### Bug fixes

> - Fix the bug when upstream database connection anomaly which may have a chance to lost data

Reviewer: ```suggestion
- Fix the bug that has a risk of data loss when the upstream database connection is abnormal 
```
